discovery-provider/plugins/notifications/src/__tests__/mappings/tip.test.ts

```diff
@@ -66,4 +68,39 @@ describe('Tip Notification', () => {
     })
   })
 
+  test("Process email notification for tip receive", async () => {
+    await createUsers(processor.discoveryDB, [{ user_id: 1 }, { user_id: 2 }])
+    // NOTE: User tips stored as a string with 8 significant digits
+    await createUserTip(processor.discoveryDB, [{ sender_user_id: 2, receiver_user_id: 1, amount: '500000000' }])
+
+    const data: TipReceiveNotification = {
+      amount: 500000000,
+      sender_user_id: 2,
+      receiver_user_id: 1
+    }
+
+    const notifications: AppEmailNotification[] = [
+      {
+        type: 'tip_receive',
+        timestamp: new Date(),
+        specifier: '1',
+        group_id: 'tip_receive:user_id:1:slot:145421349',
+        data,
+        user_ids: [2],
```

Comment:
same q here.. if user 2 is tipping user 1 as `data` shows, why is the `TipReceiveNotification` to user 2?

Reply:
it doesn't matter bc we are testing the render method, but will fix. 
You are right though, if this tested more of the flow from the beginning it would not work